src/Symfony/Component/Messenger/Stamp/HandledStamp.php

```diff
@@ -23,7 +23,7 @@
  *
  * @experimental in 4.3
  */
-final class HandledStamp implements StampInterface
+final class HandledStamp implements NonSendableStampInterface
```

Comment:
Hmm, are we sure this shouldn’t be sent? I’m thinking of the case where there are multiple handlers for a single message and only one fails. On retry, the system is currently smart enough to only retry the one, failed handler. I think it uses this.

Reply:
Good catch.  I explained that in the phpdoc.

Reply:
When combined with the DoctrineTransactionMiddleware this didn't work: #32154